graphein/protein/visualisation.py

```diff
@@ -47,6 +49,101 @@
     )
 
 
+"""
+TODO: Functino that gets ``min`` and ``max`` values in a graph for a given feature so that we can scale / offset to > 0
+TODO: should feature `distance` actually contain the site itself i.e. in the string?
+"""
+def _node_feature_func(
```

Comment:
This is really elegant work - thanks a lot!

Having taken a closer look at the code I wonder if we can be even smarter here. Given the rest of Graphein is quite functional in style, we could perhaps refactor out the sizing functions into atomic functions & then allow users to pass in either a function of their own or a string which would dispatch one the lambdas/more complicated functions you've implemented here. I think this provides the most flexibility whilst supporting all the great features you've added here. What do you think?